src/providers/cloudstack/network.rs

```diff
@@ -24,10 +23,7 @@ pub struct CloudstackNetwork {
 impl CloudstackNetwork {
     pub fn try_new() -> Result<CloudstackNetwork> {
         let server_address = CloudstackNetwork::get_dhcp_server_address()?;
-        let client = retry::Client::try_new()?
-            .initial_backoff(Duration::from_secs(1))
-            .max_backoff(Duration::from_secs(5))
-            .max_attempts(10);
```

Comment:
Are we dropping the backoffs here on purpose?

Reply:
These lines were in fact just setting again the default values:
https://github.com/coreos/afterburn/blob/2fc09f15f48908d2b28ff3e3d898e910eea60403/src/retry/mod.rs#L35-L37